browse/scripts/main.py

```diff
@@ -36,12 +37,10 @@
 f.gen_out("../../database/db.csv", "../alpha/z.md", "alphabet", "z")
 
 # topics
-f.gen_out("../../database/db.csv", "../topics/places.md", "topic", "places")
-f.gen_out("../../database/db.csv", "../topics/science.md", "topic", "science")
-f.gen_out("../../database/db.csv", "../topics/daily.md", "topic", "daily")
-f.gen_out("../../database/db.csv", "../topics/home.md", "topic", "home")
-f.gen_out("../../database/db.csv", "../topics/furniture.md", "topic", "furniture")
-f.gen_out("../../database/db.csv", "../topics/technology.md", "topic", "technology")
+tp = TopicsParser()
+topics = tp.gen_topics("../../database/db.csv")
+for topic in topics:
+    f.gen_out("../../database/db.csv", '../topics/{}.md'.format(topic), "topic", "places")
```

Comment:
shouldn't this be -

`f.gen_out("../../database/db.csv", '../topics/{}.md'.format(topic), "topic", '{}'.format(topic))`

Reply:
otherwise the filter is hardcoded to "places" tag.

Reply:
Yup you are right I did not notice that, should be fixed now.

Reply:
Thanks 😊